src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
+@import 'variables';
+@import 'calendar';
+
+* {
   margin: 0;
+  font-family: $font-family;
+  box-sizing: border-box;
 }
```

Comment:
You shouldn't use * selector for margin and padding, because it impacts the performance.

But you will probably have the same box-sizing for all your elements so let's use a selector which specializes in this.